### PR TITLE
DOMA-1687 resident ticket access fixes

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
The mobile app moves from `createResidentTicket` to` createTicket`. Because of this, the binding logic `client`,` clientName`, `clientPhone` and` clientEmail` disappeared when a ticket was sent by a resident.

Wrote this logic for `Ticket` and adjusted the access rights for the `Ticket` scheme

change `makeClientWithResidentAccessAndProperty` logic, now user with resident access is not the creator of the organization in which he is a resident

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/property/utils/testSchema/index.js [hunk 1]
```diff
@@ -58,10 +59,17 @@ async function makeClientWithProperty (includeFlat = false) {
...
-    const userClient = await makeClientWithProperty()
-    await addResidentAccess(userClient.user)
-    return userClient
+async function makeClientWithResidentAccessAndProperty () {
+    const clientWithOrganization = await makeClientWithProperty()
+    const client = await makeClientWithResidentUser()
+    const organization = clientWithOrganization.organization
+    const property = clientWithOrganization.property
+
+    await addResidentAccess(client.user)
```
Comment (toplenboren) [inline apps/condo/domains/property/utils/testSchema/index.js:68]: I think that line 64 will already add the residentAccess to the client.user

File: apps/condo/domains/property/utils/testSchema/index.js [hunk 2]
```diff
@@ -58,10 +59,17 @@ async function makeClientWithProperty (includeFlat = false) {
...
+    client.organization = organization
+    client.property = property
```
Comment (toplenboren) [inline apps/condo/domains/property/utils/testSchema/index.js:70]: I think you'd also need to create the Resident model, which would be connected to this organization and this property

Comment (nomerdvadcatpyat) [reply apps/condo/domains/property/utils/testSchema/index.js:70]: There will be a lot of tests to change, I added TODO in `b66c214`

File: apps/condo/domains/ticket/schema/Ticket.test.js [hunk 3]
```diff
@@ -60,6 +64,219 @@ describe('Ticket', () => {
...
+
+    test('resident: cannot create Ticket without unitName', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const userClient = await makeClientWithResidentAccessAndProperty()
+        const unitName = faker.random.alphaNumeric(5)
+        await createTestResident(admin, userClient.user, userClient.organization, userClient.property, {
+            unitName,
+        })
+
+        await expectToThrowAccessDeniedErrorToObj(async () => {
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/Ticket.test.js:131]: It seems, like it should be a validation error, not an access-denied.

File: apps/condo/domains/ticket/schema/Ticket.test.js [hunk 4]
```diff
@@ -60,6 +64,219 @@ describe('Ticket', () => {
...
+
+        const [updatedTicket] = await updateTestTicket(userClient, ticket.id, {
+            details: newDetails,
+        })
+
+        expect(ticket.id).toEqual(updatedTicket.id)
+        expect(updatedTicket.details).toEqual(newDetails)
+    })
+
+    test('resident: cannot update his Ticket fields other than details', async () => {
```
Comment (AntonAL) [inline apps/condo/domains/ticket/schema/Ticket.test.js:189]: What about `clientName`, `cilentPhone` and other fields?

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/ticket/schema/Ticket.test.js (hunk 4 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/Ticket.test.js:189]: @AntonAL, at the moment in figma screens a `resident` can only change the `details` and `files` from `ticket`

**Pull request merged**